packages/upload-client/test/blob.test.js

```diff
@@ -74,7 +75,7 @@ describe('Blob.add', () => {
 
     /** @type {import('../src/types.js').ProgressStatus[]} */
     const progress = []
-    const multihash = await Blob.add(
+    const { multihash } = await Blob.add(
```

Comment:
we should assert the site exists, and maybe it has capability with this multihash